### PR TITLE
Implementation of FindMeetingQuery.java

### PR DESCRIPTION
**Pull request by tayyaba17**

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -18,30 +18,39 @@
...
+    }
 
-    Collection<TimeRange> availibility = Arrays.asList();
+    Collection<Event> eventTotal = new ArrayList<>();
     for (Event event: events){
-        TimeRange before = TimeRange.fromStartEnd(TimeRange.START_OF_DAY, event.getWhen().start(), false);
-        TimeRange after = TimeRange.fromStartEnd(event.getWhen().end(), TimeRange.END_OF_DAY, true);
-        availibility.add(before);
-        availibility.add(before);
+        eventTotal.add(new Event(event.getTitle(), TimeRange.fromStartDuration(event.getWhen().start(), event.getWhen().duration()),
```
Comment (peizhaoou) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:40]: Nit: break the line? The line limit for Java is 100, if I recall correctly.

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 2]
```diff
@@ -18,30 +18,39 @@
...
+    Collection<TimeRange> availability = new ArrayList<>();
+    int startTime = TimeRange.START_OF_DAY;
+    int endTime = TimeRange.END_OF_DAY;
+    for (Event event: eventTotal){
+      TimeRange before = TimeRange.fromStartEnd(startTime, event.getWhen().start(), false);
+      availability.add(before);
+      startTime = event.getWhen().end(); 
     }
-    return availibility;
+    availability.add(TimeRange.fromStartEnd(startTime, TimeRange.END_OF_DAY, true));
```
Comment (martiKallas) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:52]: Should the startTime here be endTime? Does this pass that unit test as-is?

Comment (tayyaba17) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:52]: It does, but I just updated the variable with a better name.

**Comment by tayyaba17** [comment]
My code passes all the tests. I am trying to clean it up and remove the nested for loop. I am running to issues when comparing the attendees to eventAttendees, so I went the current way just to see if it would work. I am looking into how to compare the elements inside without the for loop. When I use contains() or equals() I run into errors. If you have any suggestions, please let me know!

**Pull request merged**